aspnetcore/mvc/views/view-compilation.md

```diff
@@ -23,9 +23,9 @@ Build- and publish-time compilation of Razor files is enabled by default by the
 
 To enable runtime compilation for all environments and configuration modes:
 
-1. Install the [Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation](https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation/) NuGet package.
```

Comment:
Order is important here. Stick with the numbered list.

Reply:
We've never had anyone complain, hey I did these out of order and it was sub-optimal in VS. That's about the only time it's even possible to be sub-optimal. Folks naturally do these in order, and if they don't, it's really not a problem. Looks much better with bullets. Reverted anyway